apps/web/frontend/components/projectMemberList.tsx

```diff
@@ -25,6 +27,11 @@ export const ProjectMemberList = (props: ProjectMemberListProps) => {
       <h2 className="text-lg font-semibold text-gray-400">Project Members</h2>
       <Table>
         <TableBody>
+          <TableRow>
+            <TableCell className="flex items-center gap-2">
```

Comment:
Why do we need a flexbox here? As I understand it, this TableCell has only one form tag as children.